src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> dataFromFile = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                dataFromFile.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from this file." + e);
```

Comment:
`throw new RuntimeException("Can't read data from this file.", e);`
Also, let's add more information to this exception